src/main/java/christmas/domain/ErrorMessage.java

```diff
@@ -0,0 +1,20 @@
+package christmas.domain;
+
+public enum ErrorMessage {
+    INVALID_MENU("[ERROR] 유효하지 않은 주문입니다. 다시 입력해 주세요."),
+    INVALID_DATE("[ERROR] 유효하지 않은 날짜입니다. 다시 입력해 주세요."),
+    OVER_MAXIMUM_ORDER("[ERROR] 메뉴의 개수는 최대 20개까지 가능합니다. 다시 입력해 주세요."),
+    ONLY_DRINK("[ERROR] 음료만 주문 시, 주문할 수 없습니다. 다시 입력해주세요."),
+    INVALID_BENEFITS_AMOUNT("[ERROR] 유효하지 않은 혜택 금액입니다.");
+
+    private final String message;
+
+    ErrorMessage(String message) {
+        this.message = message;
+    }
+
+    public String get() {
+        return message;
+    }
```

Comment:
다른 분의 코드리뷰를 하면서 눈여겨봤던 방법 추천해드려요.

```suggestion
    private static final String ERROR_PREFIX = "[ERROR] ";
    
    public String getMessage() {
        return ERROR_PREFIX + message;
    }
```
이렇게 prefix를 해당 enum 클래스에 상수로 선언하고, getter 메서드에서 붙여서 반환하는 게 추후 유지보수 측면에서도 좋아보이더라구요 :)